Gopkg.lock

```diff
@@ -241,6 +241,18 @@
   revision = "d2133a1ce379ef6fa992b0514a77146c60db9d1c"
   version = "v1.1.0"
 
+[[projects]]
```

Comment:
The changes in this file should go in the other commit.

Reply:
Moved to other commit.